### ISSUE TITLE
Handle an hidden field

### ISSUE THREAD

**Issue by jeanko84** [1000hz/bootstrap-validator#333]
Hi!
There's a problem in validation with hidden input ( passed by jquery .hide() ). The submit button is activated even if the hidden fields are not populated.
Any solution?
Thanks :)

**Comment by davewasthere** [comment]
Yes, you'll need to override the default selector so it doesn't exclude hidden fields:

`$.fn.validator.Constructor.INPUT_SELECTOR = ':input:not([type="hidden"], [type="submit"], [type="reset"], button)'`

Oh, or add `data-validate="true"` to override the selector for specific fields.

**Comment by SimJoSt** [comment]
I had the same issue and added `data-validate="true"`, but it didn't work. I believe it to be a bug.
Only way to get it working was using `class="hidden"` instead of `type="hidden"`.
Similar problem in #531
